packages/flutter_tools/lib/src/commands/apk.dart

```diff
@@ -151,14 +180,41 @@ class ApkCommand extends FlutterCommand {
         help: 'Password for the entry within the keystore.');
   }
 
+  Future _findServices(_ApkComponents components) async {
+    if (!ArtifactStore.isPackageRootValid)
+      return;
+
+    dynamic manifest = _loadYamlFile(_kFlutterManifestPath);
+    if (manifest['services'] == null)
+      return;
+
+    for (String service in manifest['services']) {
+      File serviceRoot = '${ArtifactStore.packageRoot}/$service/apk';
```

Comment:
This looks like a type error.  The right-hand side is a string and the left-hand side claims it's a File.